docs/source/app-dev/daml-lf-translation.rst

```diff
@@ -13,7 +13,7 @@ Primitive Types
 
 :ref:`Built-in data types <daml-ref-built-in-types>` in Daml have straightforward mappings to Daml-LF.
 
-This section only covers the serializable types, as these are what client applications can interact with via the generated Daml-LF. (Serializable types are ones whose values can be written in a text or binary format. So not function types, ``Update`` and ``Scenario`` types, as well as any types built up from those.)
+This section only covers the serializable types, as these are what client applications can interact with via the generated Daml-LF. (Serializable types are ones whose values can exist on the ledger. So not function types, ``Update`` and ``Scenario`` types, any types built up from those, and several other restrictions.)
```

Comment:
```suggestion
This section only covers the serializable types, as these are what client applications can interact with via the generated Daml-LF. (Serializable types are ones whose values can exist on the ledger. Function types, ``Update`` and ``Scenario`` types and any types built up from these are excluded, and there are several other restrictions.)
```

Ideally I would like either an explanation or a link to an explanation of the 'several other restrictions' if they are likely to be things that the reader will actually encounter.

Reply:
#15331